server/cluster.go

```diff
@@ -213,7 +213,7 @@ func (c *RaftCluster) loadClusterInfo() (*RaftCluster, error) {
 
 	start = time.Now()
 
-	if err := c.storage.LoadRegions(c.core.PutRegion); err != nil {
+	if err := c.regionSyncer.LoadRegionsCache(c.core.PutRegion); err != nil {
```

Comment:
`regionSyncer` does not have all the regions in its cache.  we cannot do it directly with `regionSycner`. If the PD is a follower:
- we need to incrementally sync regions to `clusterinfo` .
- we also need to ensure that the regions in the cluster are as complete as possible.